CLIENT.md

```diff
@@ -0,0 +1,67 @@
+## How to build RxPaired-client and what to do with it
+
+To build it, you first need to install its dependencies.
+
+To do that, make sure that you have `npm` installed and this repository cloned.
+Then in the root directory, open a terminal and type:
+
+```sh
+npm install
+```
+
+You have to create a `.npmrc` file in the root directory.
```

Comment:
Maybe now we'll need to add something like "if not already done", as all parts now rely on that same config file

Reply:
I added it